harpy/_validations.py

```
@@ -73,98 +73,104 @@ def validate_input_by_ext(inputfile, option, ext):
         row = 1
         badrows = []
         badlens = []
         if sorted(headersplt) != correct_header:
-            culprits = [i for i in headersplt if i not in correct_header]
-            print_error("invalid column names", f"Parameter file [bold]{parameters}[/bold] has incorrect column names. Valid names are:\n[green bold]model usebx bxlimit k s ngen[/green bold]")
+            invalid_col = [i for i in headersplt if i not in colnames]
+            missing_col = [i for i in colnames if i not in headersplt]
+            errtext = []
+            culprit_text = []
+            if invalid_col:
+                errtext.append(f"\n  - invalid column names")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)